README.md

```diff
@@ -13,27 +13,30 @@ Beginning with the next version (0.9.0), **JWST requires Python 3.5 and above**.
 
 Installing
 ----------
-Releases are made on the **stable** branch. To install the latest release:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+To install the latest stable version of the library, we recommend using [conda](https://conda.io/docs/index.html) and
+the [astroconda](https://astroconda.readthedocs.io) channel:
 
-    % cd jwst
+    conda config --add channels http://ssb.stsci.edu/astroconda-dev
+    conda install jwst
```

Comment:
Is jwst available from the astroconda-dev channel? To date we've been using conda-dev.